2-mandatory/6-journey-planner.js

```diff
@@ -20,11 +20,13 @@
 function checkCodeIsThere(stringText) {
```

Comment:
There's a problem in this fuction that prevents the tests from running; you've forgotton a closing } for the first if statement.